gitbootcamp.rst

```diff
@@ -248,3 +248,51 @@ Pull requests can be accepted and merged by a Python Core Developer.
       * rebased
 
 3. Press the ``Confirm squash and merge`` button.
+
+
+Editing a Pull Request Prior to Merging
+---------------------------------------
+
+When a pull request submitter has enabled the `Allow edits from maintainers`_
+option, Python Core Developers may decide to make any remaining edits needed
+prior to merging themselves, rather than asking the submitter to do them. This
+can be particularly appropriate when the remaining changes are bookkeeping
+items like updating ``Misc/ACKS`` and ``Misc/NEWS``.
+
+.. _Allow edits from maintainers: https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/
+
+To edit an open pull request that targets ``master``:
+
+1. At the bottom of the pull request page, click the "command line instructions"
+   link (next to the more prominent ``Squash and merge`` button).
+
+2. The given instructions are *not* the right instructions for CPython (as they
+   assume you will be pushing directly to the CPython repository, rather than
+   pushing back to the contributor's PR branch), but they include the relevant
+   repository and branch names::
+
+      $ git checkout -b <contributor>-<branch_name>
```

Comment:
Perhaps:

2. Do **not** use the GitHub "command line instructions" as displayed since they
   are not correct for CPython's workflow (as they assume pushing to the 
  CPython repository, rather than correctly pushing back to the contributor's 
  PR branch). Inspect the displayed command, 
  `git pull https://github.com/<contributor>/cpython.git <branch_name>`,
  and make a note of the `<contributor>` and `<branch_name>` since you
  will use them in following steps.